doc/tech/glossary.md

```diff
@@ -50,8 +50,8 @@ To help guide the reader, this glossary provides an introduction to the terminol
 - **NFVI Software Profile (NFVI SW Profile)**: defines the behaviour, capabilities and metrics provided by an NFVI Software Layer
```

Comment:
Suggest changing `NFVI Software Profile` to `Cloud Infrastructure Software Profile`

and

`NFVI Software Layer` to `Cloud Infrastructure Software Layer`

Reply:
done